SECURITY.md

```diff
@@ -19,6 +28,10 @@ This policy does not apply to:
 
 * Archived repositories;
 * End-user documentation and educational or "help" materials (e.g., [Tari Labs University](https://tlu.tarilabs.com/));
+* Example or test code.
+* Proof-of-concept or demonstration applications.
+* Code marked `NOT READY FOR PRODUCTION`, unless the code is demonstrably used in production applications, and those 
```

Comment:
```suggestion
* Code or repositories indicated not to be suitable for production, unless the code is demonstrably used in production applications, and those 
```
Presumably the exact wording is not required?

Reply:
For example, the new [Triptych](https://github.com/tari-project/triptych) repository is experimental and would be out of scope, but has a differently-worded [scary warning](https://github.com/tari-project/triptych?tab=readme-ov-file#warning).

Reply:
Correct. Exact wording is not necessary

Reply:
I've reworded to make this explicit.